build.sbt

```diff
@@ -116,6 +117,8 @@ lazy val ironmq = alpakkaProject("ironmq", "ironmq", Dependencies.IronMq)
 
 lazy val jms = alpakkaProject("jms", "jms", Dependencies.Jms, parallelExecution in Test := false)
 
+lazy val json = alpakkaProject("json", "json", Dependencies.Json)
```

Comment:
I wonder if the module should be called `jsonStreaming` to leave room for other nifty things with Json.